quail/processes/wps_climdex_rxnday.py

```diff
@@ -60,10 +69,26 @@ def __init__(self):
             status_supported=True,
         )
 
+    def rxnday_func(self, ci, num_days, freq, center_mean_on_last_day):
+        climdex = get_package("climdex.pcic")
+
+        if num_days == 1:
+            return climdex.climdex_rx1day(ci, freq)
+
+        elif num_days == 5:
+            return climdex.climdex_rx5day(ci, freq, center_mean_on_last_day)
```

Comment:
While it probably won't come up it could be helpful to have an `else` case that throws an error.

Reply:
@nikola-rados It throws `"InvalidParameterValue"` because of the `allowed_values`, but I could add something else?

Reply:
That should be enough then!